integration_test.go

```diff
@@ -206,7 +288,7 @@ func TestExecutionClientLifecycle(t *testing.T) {
 }
 
 func TestExecutionClient_InitChain_InvalidPayloadTimestamp(t *testing.T) {
```

Comment:
optimization: can move this test into a subtest of the above lifecycle test, to avoid spinning up the reth containers for multiple tests